app/api/getQueryData/route.ts

```diff
@@ -0,0 +1,16 @@
+import { NextResponse } from "next/server";
+import { kv } from "@vercel/kv";
+import { CachedQueryResult } from "@/shared/types/salesforceTypes";
+export async function GET(req: Request) {
```

Comment:
questionable REST naming convention

This should be like `/api/query/route.ts` with a `GET` request

Reply:
But also from a higher level implementation perspective, you shouldn't be exposing an endpoint that can just arbitrarily query the KV store at will -t his is a horrible security philosophy